Core/COBranch.m

```diff
@@ -1045,7 +1049,6 @@ - (BOOL)setCurrentNode: (id <COTrackNode>)node
 {
 	INVALIDARG_EXCEPTION_TEST(node, [node isKindOfClass: [CORevision class]]);
 	[self setCurrentRevision: (CORevision *)node];
-	[self didUpdate];
```

Comment:
It hasn't been truly removed but moved to -setCurrentRevisionSkipSupportsRevertCheck:, so the notifications are properly posted even when -setCurrentRevision: is called. For example, -undo calls -setCurrentRevision:.